docs/gateway-api/gateway-api-v1.md

```diff
@@ -164,7 +160,11 @@ In AKO Gateway API Implementation, Gateway objects corresponds to following AVI
   4. `Addresses` in a Gateway specification gets added as static ip for `Vsvip` for parent VS.
   5. Every `Rule` in `HTTPRoute` corresponds to an `EVH Child Virtual Service`, with `Match` translated to `VH match` and `Filters` translated to `HTTPPolicySet` configuration.
   6. Each `backendRefs` specification (list of backends) in a `HTTPRoute Rule` will be added as a `Pool Group`.
-  7. Each `backendRef` in a `HTTPRoute Rule` will be translated to a pool. 
+  7. Each `backendRef` in a `HTTPRoute Rule` will be translated to a pool.
+  8. If no matches are specified for a particular HTTPRoute rule, a childVS with path as `/` in VHmatch will be created and attached to the parentVS corresponding to that rule. Any request prefixed with `http://x.x.x.x/` which does not match any of the HTTProute rule will land on this  childVS and will be sent to the backend.
```

Comment:
Instead of adding this as a new point. Can we extend it as part of point 5. We just need to say how objects are mapped. I think we should not reveal in different scenario what AKO does.

Reply:
Same for remaining points. Try to give mapping to user if not already given. We don't require to explain what AKO does in different scenario in mapping section.
We can create separate section for wildcard handling and define what different combinations are supported by AKO.

Reply:
Done